NOTICES.md

```diff
@@ -545,29 +616,33 @@ https://fluentvalidation.codeplex.com/
     LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.   
 
 ## HangFire 
-http://hangfire.io/  
-
-    An easy and reliable way to perform fire-and-forget, delayed and recurring long-running, short-running, CPU or I/O intensive tasks inside ASP.NET application. No Windows Service / Task Scheduler required. Even ASP.NET is not required. 
-    Copyright © 2013-2016 Sergey Odinokov. 
-    Hangfire is an Open Source project licensed under the terms of the LGPLv3 license. Please see http://www.gnu.org/licenses/lgpl-3.0.html for license text or COPYING.LESSER file distributed with the source code. 
-    Some subscriptions allow to use Hangfire under the proprietary license, allowing private forks and modifications of Hangfire. Please see http://hangfire.io/pricing/ for more details. You can find the End User License Agreement text in the EULA file distributed with the source code. 
-    HTTPSimulator  
-    https://www.nuget.org/packages/HttpSimulator/  
-    Copyright © 2011 Ferron R Smith All rights reserved.  
-    Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:  
-    1.	Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.  
-    2.	Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.  
-    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY  
-    EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF  
-    MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL  
-    THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,  
-    SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,  
-    PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS  
-    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.  
+    http://hangfire.io/  
+
+	License
+	========
+
+	Copyright Â© 2013-2020 Sergey Odinokov.
```

Comment:
probably need to fix this line and remove the weird character

Reply:
Done . Thanks